tutorials/Desktop_Components/README.md

```diff
@@ -90,6 +90,8 @@ fun TextBox(text: String = "Item") {
 }
 ```
 
+Scrollbars can be moved by dragging the bars and using the mouse wheel or the touchpad. Horizontal scrolling with the mouse wheel can be performed by side-clicking the whell or by holding down `Shift`.
```

Comment:
Sure, looks good. But there is a typo: whell vs. wheel.

Reply:
I already fixed it)